### ISSUE TITLE
Use less valueable application partition space for the app

### ISSUE THREAD

**Issue by GoogleCodeExporter** [billycris007/open-gpstracker#112]
```
Many large tracks in OGT make a noticable dent in the available app space.
Use less or be flexible on and allow the user to move data to and from the
partition... 
```

Original issue reported on code.google.com by `rcgr...@gmail.com` on 2 Jun 2010 at 8:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `rcgr...@gmail.com` on 5 Jun 2010 at 3:17
- Added labels: **Milestone-Release1.2**
- Removed labels: **Milestone-Release0.9**

**Comment by GoogleCodeExporter** [comment]
```
Available on 2.2 devices
```

Original comment by `rcgr...@gmail.com` on 7 Aug 2011 at 12:00
- Changed state: **Fixed**
